### DISCUSSION TITLE
Aclaración duda cupones de atraso

### DISCUSSION THREAD

**Discussion by Cristian1754** [IIC2133-PUC/2024-1#173] [Tarea 2] [answered]
Tengo una duda con respecto a los cupones, tengo planeado utilizar mis dos dias, pero en el caso de que al final solo necesite 1, se puede modificar la respueta?, y si es asi, hasta cuando se puede modificar?

**Comment by sachondo7** [comment, marked as answer]
Hola, termina tu tarea tranquilo y cuando la termines puedes rellenar el form. El form se cierra bien despues, asique si terminas usando un día, llénalo ese día que hiciste el último commit y pones que solo utilizaste un día. Si usaste los 2, ahí pones que usaste los 2
se entiende? 
saludos!
